frontend/src/static/js/v_summary.js

```diff
@@ -420,6 +421,7 @@ window.vSummary = {
           const commit = commits.shift();
           week.insertions += commit.insertions;
           week.deletions += commit.deletions;
+          commit.commitResults.forEach((commitResult) => week.commitResults.push(commitResult));
```

Comment:
I am not so sure if there will be only one commit results in `commit.commitResults`, please check. If so, we do not need `forEach`.

Reply:
There might be cases with more than one commit results.

<img width="197" alt="Screen Shot 2019-06-10 at 11 10 49 AM" src="https://user-images.githubusercontent.com/32864116/59170350-af96bb80-8b70-11e9-973f-91ea4fedbd63.png">